documentation/code-gen/configure-python-sdk.md

```diff
@@ -0,0 +1,219 @@
+# Readme Configuration Guide for Python SDK
+This file describe how to configure readme files to make it avaiable for Python SDK code generation.
+
+## Common Configuration
+Configure basic package information.
+
+### Basic Information
+Configure package title/description/tag.
+~~~~
+// file: readme.md
+
+``` yaml
+title: xxxxConfigurationClient
+description: xxxx Configuration Client
+openapi-type: arm
+tag: package-xxxx-xx-xx
+```
+~~~~
+
+### tag
+A tag can contains a bunch of swagger files which are used to generate the SDK. For Example:
```

Comment:
Should we add some name rule for tag to unify it?

Reply:
yes, @akning-ms , I think we can recommend it as: package-yyyy-mm-dd[-xxxx].  where the xxxx can be 'preview' or 'only'.  I will add it in this PR.